### ISSUE TITLE
0.113 changes

### ISSUE THREAD

**Issue by sagitt** [home-assistant-community-themes/midnight#22]
not work very well on 0.113

card bordes.
![Schermata 2020-07-26 alle 22 47 05](https://user-images.githubusercontent.com/169086/88489027-f1abd800-cf91-11ea-8db7-2f2c1644a8ac.png)

**Comment by pho3nixf1re** [comment]
Please see fix in #21 . I am waiting on approval form @maartenpaauw to merge as I do not have the ability to override that requirement.

**Comment by pho3nixf1re** [comment]
Duplicate of #18

**Comment by pho3nixf1re** [comment]
The fix has been merged. Once you update this should be resolved.
